docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -178,6 +178,33 @@ How to **fix** it:
 2. **Scale up distributors**<br />
    Scaling up distributors will lower the number of in-flight push requests per distributor.
 
+### MimirIngesterReachingInflightPushRequestLimit
+
+This alert fires when the `cortex_ingester_inflight_push_requests_summary` per ingester instance limit is enabled and the actual number of in-flight push requests is approaching the set limit. Once the limit is reached, push requests to the ingester will fail (5xx) or 429 for new requests, while existing in-flight push requests will continue to succeed.
+
+In case of **emergency**:
+
+- If the actual number of in-flight push requests is very close to or already at the set limit, then you can increase the limit via CLI flag or config to gain some time
```

Comment:
Increasing via CLI flag will cause rollout of ingesters. One can increase the limit using the runtime config file instead, which is reloaded by ingesters without requring restart.

I would say that increasing the limit that exists to protect ingester against running out of memory doesn't sound very smart to me, and may cause more troubles. It really depends on the limit and if ingesters have headroom to accept more requests. It may be better to add more ingester replicas instead, but that may require changes to shuffle sharding configuration for tenants.